analyzere/requestor.py

```diff
@@ -33,21 +33,28 @@ def handle_api_error(resp, code):
         raise errors.ServerError(message, body, code, json_body)
 
 
-def request(method, path, params=None, data=None, auto_retry=True):
+def request(method, path, params=None, headers={}, data=None,
+            auto_retry=True):
```

Comment:
`{}` will persist for all calls to the function, use `None` as the default instead